build: make: No rule to make target 'vendor/.../gogo.proto', needed by 'bin/.go_protobuf_sources'.

Running `make build` in a fresh `./scripts/azworker.sh`.

```
$ make build
GOPATH set to /home/tschottdorf/go
Running make with -j16
Running make with -j16
build/node-run.sh -C ./pkg/ui yarn install
git submodule update --init
installing githooks/post-rewrite
installing githooks/commit-msg
installing githooks/post-checkout
installing githooks/prepare-commit-msg
installing githooks/post-merge
make: *** No rule to make target 'vendor/github.com/gogo/protobuf/gogoproto/gogo.proto', needed by 'bin/.go_protobuf_sources'.  Stop.
```

--- a/Makefile
+++ b/Makefile
@@ -1022,6 +1022,8 @@ CPP_SOURCES_CCL := $(subst $(PKG_ROOT),$(CPP_PROTO_CCL_ROOT),$(CPP_PROTOS_CCL:%.
 
 UI_PROTOS := $(UI_JS) $(UI_TS)
 
+$(GOGOPROTO_PROTO): $(SUBMODULES_TARGET)
+
 $(GO_PROTOS_TARGET): $(PROTOC) $(GO_PROTOS) $(GOGOPROTO_PROTO) $(BOOTSTRAP_TARGET) bin/protoc-gen-gogoroach
 	$(FIND_RELEVANT) -type f -name '*.pb.go' -exec rm {} +
 	set -e; for dir in $(sort $(dir $(GO_PROTOS))); do \

